### ISSUE TITLE
Sliver server and client issue while generating payloads

### ISSUE THREAD

**Issue by jeevanachandra** [BishopFox/sliver#902]
**Describe the bug**
RPC error while generating payloads from the sliver windows server.exe

**To Reproduce**
1. Download the release of sliver-server and client .exe or generate them by doing a git clone and by using the make command for windows.
2. Run sliver-server.exe from cmd as Admin.
3. try to generate a binary from the server.

**Expected behavior**
A exe payload should be geerated.

**Screenshots**
![image](https://user-images.githubusercontent.com/49685254/191670105-0f217995-455c-4acd-8228-2204253dbb8a.png)

**Desktop (please complete the following information):**
 - OS:Windows11 or 10

**Additional context**

**Comment by moloch--** [comment]
In the mean time, use the linux server.

**Comment by payloads45** [comment]
> In the mean time, use the linux server.

![image](https://user-images.githubusercontent.com/111106697/192156686-4d669c9b-588a-4a46-84cd-f920942ceb1f.png)

**Comment by moloch--** [comment]
This is an issue with the `generate info` logic

**Comment by analyserdmz** [comment]
Hello, thanks for this awesome project! Any progress on this issue?

**Comment by rkervella** [comment]
> Hello, thanks for this awesome project! Any progress on this issue?

If it's not closed then it's not fixed :)

**Comment by moloch--** [comment]
We'll get around to it, it's just neither of us use Windows so fixes tend to be a bit slower :)

**Comment by Boberttt** [comment]
I've been having the same issue, anyway I can manually compile the payload?

**Comment by moloch--** [comment]
Not really, the easiest workaround if you only have a windows machine is WSL or a Linux VM. We don't use Windows so fixing this issue isn't a priority but we'll eventually get around to it, just been busy :)
